applications/Shopbring.md

```diff
@@ -0,0 +1,320 @@
+* **Shuchao He**: Cryptographic algorithm engineer. He is now responsible for the development of the underlying cryptographic library of openwallet, and supports the implementation of cryptographic algorithms such as ECC, Hash, and transaction verification for mainstream blockchains.
+* **Erfeng Ma**: Dev Ops. He is now responsible for server operation and maintenance.
+
+### Contact
+
+* **Contact Name:** Zhiquan Mai
+* **Contact Email:** zhiquan911@gmail.com
+* **Website**: https://www.openwallet.cn
+
+### Legal Structure
+
+Now Individual. We will establish the company after we get the grants, and we will supplement the corporate information at that time.
+
+### Team's experience
+
+Our team has developed the [openwallet](https://github.com/blocktree) framework and redefines a wallet system model compatible with multiple blockchains. At the same time, we have build an openwallet developer platform, which integrates 70+ blockchain protocols, which greatly reduces the cost of developers and users.
+
+Our team also uses the [openwallet developer platform](https://www.openwallet.cn) to provide enterprises with blockchain-related development services, such as: cryptocurrency wallets, enterprise-level cryptocurrency management systems, multi-chain multisig wallet systems.
+
+### Team Code Repos
+
+* https://github.com/blocktree
+* https://github.com/zhiquan911
+* https://github.com/godaddy-x
+* https://github.com/fanyinghao
+* https://github.com/Brochao
+
+### Team LinkedIn Profiles
+
+* https://www.linkedin.com/in/zhiquan911
+* https://www.linkedin.com/in/shadowsick
+* https://www.linkedin.com/in/kencani
+* https://www.linkedin.com/in/fanyinghao
+* https://www.linkedin.com/in/heshuchao
+* https://www.linkedin.com/in/maerfeng
+
+## Development Roadmap :nut_and_bolt: 
+
+### Overview
+
+* **Total Estimated Duration:** 3 months.
+* **Full-time equivalent (FTE):** 5 FTE.
+* **Total Costs:** 23K DAI.
+
+### Milestone 1 UI design and System module definition
+
+* **Estimated Duration:** 1.5 month
+* **FTE:**  2
+* **Costs:** 11K DAI
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | We write complete program flow description and system module interface definition. |
+| 0c. | Testing Guide | We will deploy a POA consensus Shopbring network, which can call system modules through polkadot.js. |
+| 0d. | Article/Tutorial | We will write an article or tutorial that explains the work done as part of the grant. |
+| 1. | Substrate module: Generic Asset | Programming language: `Rust`. This module aims to manage multiple cross-chain assets. It has the following functions: `register_generic_assets`, `deregister_generic_assets`, `mint_generic_assets`, `burn_generic_assets`, `transfer_assets`. |
```

Comment:
Have a look at the [Generic Token Dealer Pallet](https://github.com/subdarkdex/pallet-generic-token-dealer), perhaps you could re-use their work and make your lives easier :)

Reply:
Thank you, let me compare which is more suitable.